src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -1001,6 +1059,11 @@ mgp_error mgp_list_copy(mgp_list *list, mgp_memory *memory, mgp_list **result) {
 
 void mgp_list_destroy(mgp_list *list) { DeleteRawMgpObject(list); }
 
+mgp_error mgp_list_contains_deleted(mgp_list *list, int *result) {
+  *result = ContainsDeleted(list);
+  return mgp_error::MGP_ERROR_NO_ERROR;
```

Comment:
Why not use WrapExceptions?

Reply:
I didn’t envision any case where this function would throw an exception, but I can add it no problem